tools/src/tensil/tools/Main.scala

```diff
@@ -41,14 +41,14 @@ object Main extends App {
       .text("Tensil architecture descrition (.tarch) file")
 
     opt[Seq[String]]('o', "output")
-      .valueName("<names>")
+      .valueName("<name>,...")
       .action((x, c) => c.copy(outputNodes = x))
       .text("Optional list of output nodes, defaults to \"Identity\"")
 
-    opt[Int]('b', "batch")
-      .valueName("<integer>")
-      .action((x, c) => c.copy(inputBatchSize = x))
-      .text("Optional size of input batch, defaults to 1")
+    opt[String]('i', "input-shapes")
+      .valueName("<name>[<dim>,...],...")
```

Comment:
should there be a space between `<name>` and `[<dim>,...]`?

Reply:
The space is optional in the parser. I added spaces in argument description for better looks.